File: source/back/BankInformationSystem.Data/Entities/BirthInfo.cs
using System;

namespace BankInformationSystem.Data.Entities
{
    public class BirthInfo
    {
        public int Id { get; set; }
        
        public string PlaceOfBirth { get; set; }
        
        public DateTime DateOfBirth { get; set; }

        public int CustomerId { get; set; }
        
        public virtual Customer Customer { get; set; }
    }
}

File: source/front/src/models/entities/BirthInfo.ts
import { Stateful, action } from "reactronic"

export interface IBirthInfo {
  placeOfBirth: string
  dateOfBirth: string // DateTime
}

export class BirthInfo extends Stateful {
  placeOfBirth: string
  dateOfBirth: string

  constructor(birthInfo: IBirthInfo) {
    super()
    this.placeOfBirth = birthInfo.placeOfBirth
    this.dateOfBirth = birthInfo.dateOfBirth
  }

  @action
  setPlaceOfBirth(value: string): void {
    this.placeOfBirth = value
  }

  @action
  setDateOfBirth(value: string): void {
    this.dateOfBirth = value
  }
}
